src/model/add-dropcaps.ts

```diff
@@ -0,0 +1,68 @@
+import { JSDOM } from 'jsdom';
```

Comment:
Yeah, think it's much cleaner to have each 'mod' in a separate file. Does 'model' describe the folder well? Maybe `enhanceCAPI` or `hacks` - model feels a little too generic and I think it's worth have the folder name descriptive.

Reply:
Maybe if we call it `hacks` it's a clear message that we want to remove this code at a later stage